db/migrate/20210216160312_remove_results_fails_and_zeros.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveResultsFailsAndZeros < ActiveRecord::Migration[6.0]
```

Comment:
Merge this with the other migration (you only need one since these are logically-related changes).